src/data/finset/lattice.lean

```diff
@@ -249,6 +249,14 @@ le_antisymm
 lemma sup_id_eq_Sup [complete_lattice α] (s : finset α) : s.sup id = Sup s :=
 by simp [Sup_eq_supr, sup_eq_supr]
 
+lemma sup_id_set_eq_sUnion {α} (s : finset (set α)) :
+  s.sup id = ⋃₀(↑s) :=
+sup_id_eq_Sup _
+
+@[simp] lemma sup_set_eq_bUnion {α β : Type*} {s : finset α} {f : α → set β} :
+  s.sup f = ⋃ x ∈ s, f x :=
+sup_eq_supr _ _
```

Comment:
You seem not to have applied the suggestion I made in https://github.com/leanprover-community/mathlib/pull/12459#discussion_r821054916 to remove the `α β` and make `s` and `f` explicit arguments

Reply:
I am sorry. I have difficulties tracking what modifications are made and which aren't.

Reply:
Now, it's done.